src/components/blog-page/BlogCardList.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { FC } from "react";
+
+import "./css/blog-card-list.css";
+
+interface BlogCardListProps {
+  articles: {
+    id: number;
+    frontmatter?: {
+      title: string;
+    };
+    fields?: {
+      slug: string;
+    };
+  }[];
+}
+
+export const BlogCardList: FC<BlogCardListProps> = ({ articles }) => {
+  const listItems = articles.map((article) => {
+    return (
+      <li className="blog-card-list__card" key={article.id}>
+        <a href={`/blog${article.fields?.slug}`}>{article.frontmatter?.title}</a>
```

Comment:
we should use a gatsby `<Link>` here to benefit from gatsby routing for performance reasons